bin/ka-clone

```diff
@@ -197,13 +218,6 @@ def install_pre_push_lint_hook():
     _cli_log_step_success("Added pre-push linting hook")
 
 
-def _git_dir():
-    """Retrieve the GIT_DIR from the system."""
-    return subprocess.check_output(
-        ["git", "rev-parse", "--git-dir"]
-    ).decode('utf-8').rstrip()
-
-
```

Comment:
moved this up since the rest of our "get the path/dir" functions are towards the top